packages/jest-jasmine-adapter/src/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+const {addMatchers} = require('jest-matchers');
+
+// A single place to store everything jasmine related. (hacks, patches,
+// backward compatibility functions, etc.)
+const setup = () => {
+  const jasmine = global.jasmine || require('jasmine');
```

Comment:
yeah... my plan was to remove jasmine2 runner and replace it with jest runner.
This means there will be no global `jasmine` at all.
and for backward compatibility we'll use this `jest-jasmine-adapter` that will take pieces from jasmine (like utils and all that stuff) and provide similar API.
But of course it's overengineering and we can always do it later :)